packages/react-ui/src/hooks/platform-hooks.ts

```diff
@@ -26,6 +38,7 @@ export const platformHooks = {
       refetch: async () => {
         await query.refetch();
       },
+      setCurrentPlatform,
```

Comment:
nit: inline the definition there and extract "platform" as variable and reuse in the above 2 methods